nsv13/code/modules/overmap/types/nanotrasen.dm

```diff
@@ -227,6 +251,21 @@
 	armor = list("overmap_light" = 95, "overmap_medium" = 75, "overmap_heavy" = 25)
 	overmap_deletion_traits = DAMAGE_STARTS_COUNTDOWN
 
+/obj/structure/overmap/nanotrasen/serendipity/starter
+	role = MAIN_OVERMAP
+	obj_integrity = 900
+	max_integrity = 900
+	integrity_failure = 900
+	starting_system = "Staging" //Required for all player ships
+	armor = list("overmap_light" = 50, "overmap_medium" = 40, "overmap_heavy" = 50)
+	overmap_deletion_traits = DAMAGE_STARTS_COUNTDOWN
+
+/obj/structure/overmap/nanotrasen/serendipity/starter/apply_weapons()
+	weapon_types[FIRE_MODE_GAUSS] = new /datum/ship_weapon/gauss(src)
+	weapon_types[FIRE_MODE_PDC] = new /datum/ship_weapon/pdc_mount(src)
+	weapon_types[FIRE_MODE_TORPEDO] = new /datum/ship_weapon/torpedo_launcher(src)
```

Comment:
VLS function under FIRE_MODE_AMS, not FIRE_MODE_TORPEDO. You'd want weapon_types[FIRE_MODE_AMS] = new /datum/ship_weapon/vls(src).